test/index.test.ts

```diff
@@ -468,6 +468,23 @@ describe("ofetch", () => {
     ).rejects.toThrow("error in onResponseError");
   });
 
+  it("hook modifications", async () => {
+    // onRequest
+    expect(
+      await $fetch(getURL("/echo"), {
+        method: "POST",
+        body: { num: 42 },
+        onRequest(ctx) {
+          ctx.options.headers = {
+            ...ctx.options.headers,
```

Comment:
This usage is totally wrong. `ctx.options.headers` is typed explicitly as `Headers`

Reply:
Yeah, it should be. 
Previously, it was implicitly allowed because the internals were normalized as `Headers` regarding the type. A small change in `v1.5.0` breaks this implicit behavior. Although we made no promise regarding this, I think it would be better if we could keep that without extra burden (or bumping major versions).